### PR TITLE
Fixed issues in OpenGraph meta tags.

### PR DESCRIPTION
**Pull request by ochrons**
Added a site-wide og:description.

### CODE REVIEW

File: _includes/ogp.html [hunk 1]
```diff
@@ -1,18 +1,22 @@
 {% capture header %}
 
 {% if page.title %}
-<meta property="og:type" content="article" />
+<meta property="og:type" content="website" />
```
Comment (sjrd) [inline _includes/ogp.html]: We should have `article` for news, shouldn't we?

File: _includes/ogp.html [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
+<meta property="og:description" content="{{ site.description }}" />
+{% endif %}
 
-{% capture page_url %}{{ page.url | replace:'/index.html','/' | prepend: site.url }}{% endcapture %}
-{% include meta.html property="og:url" content=page_url %}
+{% include meta.html property="og:site_name" content=site.title %}
 
-{% capture image_url %}{{ site.url | append: '/assets/img/scala-js-logo-256.png'}}{% endcapture %}
+{% capture image_url %}{{ site.url | append: '/assets/img/scala-js-logo-128.png'}}{% endcapture %}
 {% include meta.html content=image_url property="og:image" %}
```
Comment (sjrd) [inline _includes/ogp.html]: https://developers.facebook.com/docs/sharing/best-practices#images

> **Minimum Image Size**
> The minimum image size is 200 x 200 pixels. If you try to use an image smaller than this you will see an error in the URL Debugger.

File: _includes/ogp.html (hunk 1 above)
Comment (ochrons) [inline _includes/ogp.html]: IMO the "article" is more like a real blog post or a news story, with an author etc. We don't really have that.

File: _includes/ogp.html (hunk 2 above)
Comment (ochrons) [inline _includes/ogp.html]: Ah, those pesky demands :D I'll change it back.. it just looked so big in my tests :)

File: _includes/ogp.html (hunk 1 above)
Comment (sjrd) [inline _includes/ogp.html]: OK

**Comment by sjrd** [comment]
Could you squash the two commits, please?

**Comment by sjrd** [comment]
LGTM

**Pull request merged**